spring-pulsar-spring-boot-autoconfigure/src/test/java/org/springframework/pulsar/autoconfigure/SpringPulsarBootAppSanityTests.java

```diff
@@ -34,7 +34,9 @@
  *
  * @author Chris Bono
  */
-@SpringBootTest(classes = SpringPulsarBootTestApp.class)
+@SpringBootTest(classes = SpringPulsarBootTestApp.class,
+		properties = { "spring.main.banner-mode=off"
```

Comment:
The SB banner is logged very early, before the test log config file is respected apparently and it always was showing up in the output. This was the only test that was happening in and it is gone now. 

At this point the only thing we see in test output is WARN,ERROR logs.